README.md

```diff
@@ -101,11 +129,11 @@ Which of the following statements will work, assuming jQuery is loaded?
 
 Select all that apply:
 ```
-[] `$(".post").css("background", "peachpuff")`
+[x] `$(".post").css("background", "peachpuff")`
 [] `$(".post").innerHTML`
-[] `$(".post").html()`
+[x] `$(".post").html()`
 [] `document.getElementsByClassName("post")[0].innerHTML`
-[] `document.getElementsByClassName("post").innerHTML`
+[x] `document.getElementsByClassName("post").innerHTML`
 ```
```

Comment:
`document.getElementsByClassName("post")[0].innerHTML` is valid but the other is not. Does `.getElementsByClassName` return an array or a DOM object?